core-ui/src/components/Extensibility/helpers/index.js

```diff
@@ -2,6 +2,7 @@ import React, { createContext, useContext } from 'react';
 import { Trans, useTranslation } from 'react-i18next';
 import { OrderedMap } from 'immutable';
 import { last } from 'lodash';
+import { merge } from 'lodash';
```

Comment:
Could you please add the `merge` import to the one where you import `last`, please?